src/demos/datepicker/fuzzy-datepicker-demo.component.html

```diff
@@ -0,0 +1,133 @@
+<h2
+  class="sky-section-heading"
+>
+  Standard fuzzy datepicker
```

Comment:
What is "standard" about this? Using that terminology makes it sound like "recommended," which I don't think is what we mean to convey. Should "Standard" be "Template-driven"? Or something else?

Reply:
Changed to "Template-driven".